core/src/test/java/org/elasticsearch/action/search/TransportMultiSearchActionTests.java

```diff
@@ -102,14 +111,17 @@ protected void doExecute(SearchRequest request, ActionListener<SearchResponse> l
                 });
             }
         };
-        TransportMultiSearchAction action =
-                new TransportMultiSearchAction(threadPool, actionFilters, transportService, clusterService, searchAction, resolver, 10);
+
+        TransportMultiSearchAction action = new TransportMultiSearchAction(threadPool,
+                actionFilters, transportService, clusterService, searchAction, resolver, 10,
+                System::nanoTime);
```

Comment:
As far as I can tell, this is the only change in this file that was necessitated by the changes in this PR.